src/annotator/test/annotation-sync-test.js

```diff
@@ -1,21 +1,19 @@
-import EventEmitter from 'tiny-emitter';
+import { EventBus } from '../util/emitter';
 
 import AnnotationSync from '../annotation-sync';
 
 describe('AnnotationSync', () => {
-  let createAnnotationSync;
+  let emitter;
   let fakeBridge;
-  let options;
+  let createAnnotationSync;
   let publish;
   const sandbox = sinon.createSandbox();
 
   beforeEach(() => {
-    const emitter = new EventEmitter();
-    const listeners = {};
+    const eventBus = new EventBus();
+    emitter = eventBus.createEmitter();
```

Comment:
This emitter is not currently cleaned up in `afterEach`.

Reply:
Good catch, thanks!

It is fixed.